### PR TITLE
Enables ISCSI CSI Driver image

### PR DESCRIPTION
**Pull request by huffmanca**
Enables the iSCSI CSI Driver Image for stage building.

### CODE REVIEW

**Comment by msau42** [comment]
Can you run the script and also include the updated file?

**Comment by huffmanca** [comment]
@msau42 ,

I ran `hack/update-all.sh`. There were some failures on `hack/update-gopherage.sh` (due to `gopherage/cmd/html/static/zz.browser_bundle.es2015.js` not being present), but every other script executed without issue and no files were updated.

Is there another script that I should be running?

**Comment by msau42** [comment]
I meant this one that's being modified: config/jobs/image-pushing/k8s-staging-sig-storage.sh

**Comment by huffmanca** [comment]
@msau42 ,

This has been run, and the updated file included.

File: config/jobs/image-pushing/k8s-staging-sig-storage.yaml [hunk 1]
```diff
@@ -19,7 +19,7 @@ postsubmits:
       spec:
         serviceAccountName: gcb-builder
         containers:
-          - image: gcr.io/k8s-testimages/image-builder:v20210204-b06ec78
+          - image: gcr.io/k8s-testimages/image-builder:v20200901-ab141a0
```
Comment (msau42) [inline config/jobs/image-pushing/k8s-staging-sig-storage.yaml:22]: I think you'll need to update k8s-staging-sig-storage.sh with the new image-builder tag. Unfortunately the script that auto updates the image version doesn't update the script that generates this yaml.

Comment (spiffxp) [reply config/jobs/image-pushing/k8s-staging-sig-storage.yaml:22]: https://github.com/kubernetes/test-infra/pull/20960 should make autobump change the script going forward

**Comment by huffmanca** [comment]
This PR has been rebased to include the changes in https://github.com/kubernetes/test-infra/pull/20960 , and the script has been re-run.

**Comment by msau42** [comment]
/lgtm
/approve

Thanks!

**Review by spiffxp — APPROVED** [review]
/approve
/lgtm

**Pull request merged**